app/partials/target-disease.html

```diff
@@ -417,6 +417,45 @@ <h3 view-title>Evidence for {{(search.info.gene.approved_symbol || search.info.g
 
                     </div uib-accordion-group>
 
+                    <!-- Nferx -->
```

Comment:
This can't go in the public version. If there is the need for adding new datatypes and sections to the evidence page I would prefer to convert the evidence sections to plugins as we have done with the profile pages.